src/store/VisualStyleStore.ts

```diff
@@ -194,49 +195,110 @@ export const useVisualStyleStore = create(
       createContinuousMapping(
         networkId,
         vpName,
+        vpType,
         attributeName,
         attributeValues,
       ) {
         set((state) => {
           const DEFAULT_COLOR_SCHEME = ['red', 'white', 'blue']
+          const DEFAULT_NUMBER_RANGE =
+            !vpName.includes('Opacity') && !vpName.includes('opacity')
```

Comment:
Opacity is a special case because it's range is [0, 100].  This also brings up another point about having valid ranges for each visual property. Maybe this is another model change?